eta/core/storage.py

```diff
@@ -613,6 +614,47 @@ def delete(self, file_or_folder_id):
         self._service.files().delete(
             fileId=file_or_folder_id, supportsTeamDrives=True).execute()
 
+    def delete_all_files_in_folder(self, folder_id):
+        '''Deletes all the files in the Google Drive folder and retains the
+        original folder.
+
+        Args:
+            folder_id: the ID of the Google Drive folder from which the files
+                are to be deleted
+
+        Returns:
+            an empty folder with no files
+        '''
+        files = self.list_files_in_folder(folder_id)
+        for f in files:
+            filename = f["name"]
+            try:
+                with etau.Timer() as t:
+                    self.delete(f["id"])
+                    logger.info(
+                        "File '%s' deleted from '%s' (%s)", filename, folder_id,
+                        t.elapsed_time_str)
+            except Exception as e:
+                if not skip_failures:
+                    raise GoogleDriveStorageClientError(e)
+
+        logger.info("All files deleted in %s", folder_id)
+
+    def get_count_of_files_in_folder(self, folder_id):
```

Comment:
this could be implemented in one line as
```py
return len(self. list_files_in_folder(folder_id))
```

I'm not sure this is complex enough to warrant its own method, but if it stays then please:
- rename to `count_files_in_folder()` for brevity
- implement as one-liner above

Reply:
wanted to have a method to get the count of files (for example, to get the count of validation/training images in the GoogleDrive folder). will implement in the one-liner mentioned, thanks.